pkg/controller/replication/conversion.go

```diff
@@ -125,13 +125,13 @@ type conversionEventHandler struct {
 	handler cache.ResourceEventHandler
 }
 
-func (h conversionEventHandler) OnAdd(obj interface{}) {
+func (h conversionEventHandler) OnAdd(obj interface{}, isInInitialList bool) {
```

Comment:
@liggitt I think this was the only middle layer I had to fix -- IMO this fix is a lot more convenient than adding a second method and making both work